### DISCUSSION TITLE
Does EZSP support devices from KMPCIL vendor ?

### DISCUSSION THREAD

**Discussion by moodyblue** [arendst/Tasmota#15758] [Q&A] [answered]
Hi, I'm planning to acquire some of [these devices](https://www.zigbee2mqtt.io/supported-devices/#v=KMPCIL). Are they supported by EZSP (on Sonoff ZBBridge) ?

**Comment by s-hadinger** [comment, marked as answer]
In theory yes if they are following the Zigbee standard. Hard to say before actually trying.

**Comment by moodyblue** [comment]
Thanks. I've ordered one device and will report back.
